packages/component-base/package.json

```diff
@@ -21,8 +21,6 @@
     "!**/*.ts",
     "!**/*.tsx",
     "**/*.d.ts",
-    "!**/*.spec.*",
-    "!docs",
```

Comment:
I didn't leave omitting the `docs` folder from this because `component-base` is not really a component. But I can add it back in if anyone sees it might happen.